iosTests/package.json

```diff
@@ -21,7 +21,7 @@
         "@react-native-community/eslint-config": "^2.0.0",
         "@types/jest": "^26.0.24",
         "@types/react": "^18.0.21",
-        "@types/react-native": "^0.70.6",
+        "@types/react-native": "^0.70.13",
```

Comment:
I don't see a 0.70.13 for @types/react-native. See [here](https://www.npmjs.com/package/@types/react-native?activeTab=versions).

Reply:
😬 [It exists as a release](https://github.com/facebook/react-native/releases/tag/v0.70.13) and worked in my testing.

Reply:
@wmathurin [It's there](https://www.npmjs.com/package/@types/react-native/v/0.70.13).  The releases are sorted by recent downloads, not by tag name (they just happen to line up).  If you scroll down you will find `0.70.13`.

Reply:
You are right !! It's sorted in chronological order. I should have done a "search in page".